README.md

```diff
@@ -5,18 +5,22 @@ This is the official operator to deploy and manage IBM block storage CSI driver.
 
 Supported container platforms:
```

Comment:
All the changes should be consistent with driver readme

Reply:
Since the operator readme was reviewed more (and first) and is more important for users, I think the driver readme should be aligned with the operator, and not vice versa.

Regardless, they are not completely identical in 1.0.0, so if something is not aligned, you need to comment on the relevant line (this line looks valid to me)